src/docs/examples.md

```diff
@@ -0,0 +1,44 @@
+# KGCL Commands.
+
+## `NodeRename`:
```

Comment:
Link to URIs like https://w3id.org/kgcl/NodeRename

Reply:
Done